package.json

```diff
@@ -79,6 +79,7 @@
     "test": "jest test/ --verbose",
     "test:fast": "FAST_TEST=1 jest test/ --verbose",
     "debug": "node -r ts-node/register --inspect-brk ts-json-schema-generator.ts",
-    "run": "ts-node ts-json-schema-generator.ts"
+    "run": "ts-node ts-json-schema-generator.ts",
+    "prepare": "yarn build"
```

Comment:
Why is that needed? `ts-node` works without building the project.

Reply:
I'll remove it. I added that to get it building when installing directly from github in the package.json dependencies.